### DISCUSSION TITLE
Как выйти из аккаунта?

### DISCUSSION THREAD

**Discussion by ice-rider** [community/community#75555] [Codespaces] [answered]
допустим я залогинился с 1ного аккаунта,  а потом я хочу выйти и с другого аккаунта залогиниться. как это сделать?
попробовал `/sign in` но не сработало

**Comment by ice-rider** [comment]
я не исключаю что можно выйти просто удалив все данные расширения в папке вс кода но к чему такое...

**Comment by AidenJamesYt** [comment, marked as answer]
**russian answer:**
Чтобы выйти из одной учетной записи и войти в другую в Visual Studio Code, выполните следующие действия:

Щелкните значок учетной записи на панели действий, которая находится в нижнем левом углу окна. 
Visual Studio Code Activity Bar[Opens in a new window](https://code.visualstudio.com/api/ux-guidelines/activity-bar)
[code.visualstudio.com](https://code.visualstudio.com/api/ux-guidelines/activity-bar)
Visual Studio Code Activity Bar

Щелкните по своему имени пользователя.

Выберите Выйти.

Чтобы войти в другую учетную запись, снова щелкните значок учетной записи и выберите Войти.

Введите новые учетные данные и нажмите Войти.

Удаление данных расширения в папке vscode не приведет к выходу из учетной записи. Это только удалит ваши настройки и предпочтения для расширения.


**english answer:**
1 Click on the account icon in the Activity Bar, which is located at the bottom left of the window.

2 Click on your username.

3 Select Sign out.

4 To log in with a different account, click on the account icon again and select Sign in.

5 Enter your new account credentials and click Sign in.

Deleting the extension data in the vscode folder will not log you out of your account. It will only delete your settings and preferences for the extension.

**Comment by 7Chaika7** [comment]
Как удалить свой аккаунт, Зарегистрировалась, но поняла, что ошиблась.
